app/Models/WaitingRoom.php

```diff
@@ -29,6 +29,19 @@ public function users()
         return $this->belongsToMany(User::class);
     }
 
+    /**
+     * Determines if the waiting room is open for signupsOpen
+     *
+     * @return bool whether the room is open or not
+     */
+    public function isOpen()
+    {
+        $start = Carbon::parse($this->signup_start_date)->startOfDay();
+        $end = Carbon::parse($this->signup_end_date)->endOfDay();
+        $today = Carbon::now();
+        return ($today >= $start) && ($today <= $end);
```

Comment:
Hmm, are you sure? The `between` method should default to `>=` and `<=` for comparison.